OsmAnd/src/net/osmand/plus/GpxSelectionHelper.java

```diff
@@ -54,7 +56,12 @@
 	private Map<GPXFile, Long> selectedGpxFilesBackUp = new java.util.HashMap<>();
 	private SavingTrackHelper savingTrackHelper;
 	private final static Log LOG = PlatformUtil.getLog(GpxSelectionHelper.class);
+	private SelectGpxTask selectGpxTask;
```

Comment:
Field is not used anywhere

Reply:
This field is used for checking running SelectGpxTask in runSelection